### ISSUE TITLE
Is the fine-tuning needed?

### ISSUE THREAD

**Issue by pablovalle** [SHAILAB-IPEC/EO1#2]
### System Info

Hi Delin,

Thank you for this great work, I tried to create an issue on the github repo but I was unable to do so. My question is whether do I need to fine-tune the model on hugging face in order to use it on the SimplerEnv environment. If so, I don't have the needed rouserces so do you plan to share the checkpoints for the benchmark?

Thank you in advance!

### Reproduction

-

### Expected behavior

-

**Comment by DelinQu** [comment]
Hi! Thanks a lot for your interest and for reaching out.

I think one of the key differences between LLMs and robotics models is that robotic tasks inevitably require post-training. You can refer to GEN-0’s latest blog for a great discussion on this point:
https://generalistai.com/blog/nov-04-2025-GEN-0

Pretraining gives broad generalization ability across diverse tasks,
but post-training is what really improves accuracy and reliability on a specific downstream task. In fact, a purely pretrained model can sometimes perform quite poorly, and the data quality requirements for post-training are usually much lower.

So yes — some degree of fine-tuning is generally necessary if you want strong performance in SimplerEnv or other robotics benchmarks. 

We're in the CVPR duration, and will update the repo in these two weeks.

**Comment by pablovalle** [comment]
Hi! Thank you for your fast response.

“I’ll stay tuned then. I’m looking forward to testing this model on my benchmark and incorporating it into the VLAs I primarily use in my studies.

Best regards!

**Comment by DelinQu** [comment]
Thank you!

**Comment by DelinQu** [comment]
> Hi! Thank you for your fast response.
> 
> “I’ll stay tuned then. I’m looking forward to testing this model on my benchmark and incorporating it into the VLAs I primarily use in my studies.
> 
> Best regards!

Hi [pablovalle](https://github.com/pablovalle)! We have updated the models for simpler env! You can check them in [eo1-qwen25_vl-fractal](https://huggingface.co/IPEC-COMMUNITY/eo1-qwen25_vl-fractal) and [eo1-qwen25_vl-bridge](https://huggingface.co/IPEC-COMMUNITY/eo1-qwen25_vl-bridge).

**Comment by pablovalle** [comment]
Hi @DelinQu! Awasome news! Thank you very much!!!
